core/lib/globals_shared.h

```diff
@@ -212,38 +212,52 @@ typedef byte *app_pc;
 typedef void (*generic_func_t)();
 
 #ifdef WINDOWS
-#    ifndef DR_DO_NOT_DEFINE_uint64
-typedef unsigned __int64 uint64;
+#    ifndef DR_DEF_FOR_uint64
```

Comment:
I think we have to turn off DR_DO_NOT_DEFINE if the corresponding DR_DEFINE_FOR is set.  The CHECK_TYPE_SIZE commands in our top-level CMakeLists.txt will auto-enable DR_DO_NOT_DEFINE, so an external project that's using CMake and has a type coming from a header checked by CHECK_TYPE_SIZE will not be able to use this DR_DO_NOT_DEFINE feature as-is.

Reply:
done.